frontend/src/app/containers/App.js

```diff
@@ -1,23 +1,28 @@
-/* global ga */
+/* global ga, location */
```

Comment:
Generally a good practice to refer to window-scoped variables by that prefix, so `window` here and `window.location`, maybe?